src/components/Admin/ServiceSpecs/index.js

```diff
@@ -199,19 +203,19 @@ export default class ServiceSpecs extends React.Component {
         variables: form.values(),
       }).then(({data}) => {
         this.props.data.refetch()
-        this.props.store.app.setSnackbar({ 
-          msg: this.form.values()['name'] + " has been created",
+        this.props.store.app.setSnackbar({
+          msg: form.values()['name'] + " has been created",
           open: true,
-        })        
+        })
         this.closeDrawer(true)
       });
     } else {
       this.props.updateServiceSpec({
         variables: form.values(),
       }).then(({data}) => {
         this.props.data.refetch();
-        this.props.store.app.setSnackbar({ 
-          msg: this.form.values()['name'] + " has been updated",
+        this.props.store.app.setSnackbar({
+          msg: form.values()['name'] + " has been updated",
```

Comment:
Curious why you ditched the `this` ?

Reply:
just one less reference since we're already passing in the form object. do you prefer `this`?